viso_odom_calculator:
  rate: 5
  force_feedback_sigma: 1.0
  distribution_feedback_minimum_sigma: 0.1
  max_feedback_time: 0

biped_particle_odometry:
  particle_num: 30
  valid_particle_num: 20
  rate: 40
  use_imu: true
  init_sigma_x: 0.05
  init_sigma_y: 0.05
  init_sigma_z: 0.001
  init_sigma_roll: 0.001
  sigma_pitch: 0.001
  sigma_yaw: 0.05

biped_particle_odometry_iir_filter:
  cutoff: 5
  rate: 40

# use 2015-12-19-14-14-24 velocity model
viso_odometry_offset:
  use_twist_filter: false
  overwrite_pdf: true
  twist_proportional_sigma: false
  sigma_x: 0.05
  sigma_y: 0.1
  sigma_z: 0.03
  sigma_roll: 0.3
  sigma_pitch: 0.3
  sigma_yaw: 0.06
  mean_x: 0.0
  mean_y: -0.002
  mean_yaw: -0.002

biped_odometry_offset:
  use_twist_filter: true
  filter_buffer_size: 10
  calculate_covariance: true
  twist_proportional_sigma: false
  overwrite_pdf: true
  sigma_x: 0.075
  sigma_y: 0.15
  sigma_z: 0.015
  sigma_roll: 0.001
  sigma_pitch: 0.001
  sigma_yaw: 0.15
  mean_x: 0.006
  mean_y: 0.001
  mean_yaw: 0.0